dstep/translator/IncludeHandler.d

```diff
@@ -109,21 +123,43 @@ class IncludeHandler
         imports ~= "core.stdc.config";
     }
 
+    /// Makes includes that match regex filter be converted to import with prefix.
+    void setAutoImportPrefix (string prefix)
+    {
+        this.convertIncludes = true;
+        this.importPrefix = prefix;
+    }
+
+    /// Makes includes that match regex filter be converted to import with prefix.
+    void setAutoImportFilter (string filter)
+    {
+        this.convertIncludes = true;
+        this.convertableIncludePattern = regex(filter);
+    }
+
     string[] toImports ()
     {
-        auto r =  rawIncludes.map!((e) {
+        auto r =  mambo.core.Array.map!((e) {
```

Comment:
I got an ambiguity about the map in mambo and the standard library. I found a cleaner way to resolve it.